source/agora/node/Validator.d

```diff
@@ -61,6 +61,13 @@ import core.time;
 
 public class Validator : FullNode, API
 {
+    protected enum ValidatorTimers
+    {
+        NameRegistration = super.FullNodeTimers.max + 1,
```

Comment:
Don't need the `super` here.

Reply:
Thought it made it clearer (like using `this` when relevant but not required) but can remove.